elasticsearch-details/indices-rule-templates.md

```diff
@@ -478,13 +546,83 @@ The request will be allowed when all of following conditions are met:
 * all aliases of the existing template are allowed,
 * all aliases of the requested template are allowed.
   
+<details>
+  <summary>Example (click to expand)</summary>
+
+In the previous example, user `dev1` created the component template `ctemp1` with one alias `idev1`. Let's check if user `dev2` will be able to modify it:
+
+```text
+$ curl -vk -u dev2:test "http://localhost:9200/_component_template/ctemp1?pretty" -XPUT -H "Content-Type: application/json" -d \
+  '{
+     "template": {
+   	   "settings": {
+   	     "index.number_of_replicas": 0
+   	   },
+   	   "aliases": { 
+   	     "idev2": {}
+   	   }
+     }
+  }'
+
+  HTTP/1.1 403 Forbidden
+  content-type: application/json; charset=UTF-8
+
+  {
+    "error" : {
+      "root_cause" : [
+        {
+          "reason" : "forbidden",
+          "due_to" : ["OPERATION_NOT_ALLOWED"]
+        }
+      ],
+      "reason" : "forbidden",
+      "due_to" : ["OPERATION_NOT_ALLOWED"],
+      "status" : 403
+    }
+  }
+```
+
+No. And this is good behaviour, because `dev2` doesn't have an access to the alias `idev1` which the `ctemp1` has. ROR assumes, that he isn't an owner of the component template (please notice, that the same request will be allowed when a different, nonexistent component template name is used). I can assure you that `dev1` is able to modify the template (you can check if you want).
+
+</details>
+
 ---
 ### Delete a component template
 
 The request will be allowed when template does not exist OR all of the following conditions are met:
 * a template with requested name does exist,
 * all aliases of the existing template are allowed.
-  
+
+<details>
+  <summary>Example (click to expand)</summary>
+
+If you read the previous example, here you won't find anything interested. A component template can be removed only by an owner of the template. If ROR considers that a user is an owner of a requested component template, it allows to remove it. See that `dev2` is not able to remove `ctemp1`:
```

Comment:
interested -> interesting

What is an "owner" tho?

Reply:
good point. I removed "owner" and "ownership" words.